docs/source/user_guide/types.rst

```diff
@@ -0,0 +1,145 @@
+======================
+Type Support In Koalas
+======================
+
+.. currentmodule:: databricks.koalas
+
+
+This is based on Koalas 1.1.4.
+
+Koalas supports various of types by mapping them to specific types in PySpark internally.
+
+This chapter gives you an information of what types are supported and which types are not.
+
+
+Types supported in Koalas
+-------------------------
+
+A table below shows which NumPy types are matched to which PySpark types internally in Koalas.
+
+============= =======================
+NumPy         PySpark
```

Comment:
I think we don't have to expose our internal mapping. PySpark is just an internal engine in Koalas.